src/main/ruby/truffleruby/core/truffle/exception_operations.rb

```diff
@@ -113,15 +120,18 @@ def self.message_and_class(exception, highlight)
               result << "\n\e[1m#{line.chomp}\e[m"
             end
           end
+
           result
         else
           "\e[1m#{message}#{highlighted_class}\e[m"
         end
       else
-        if i = message.index("\n")
-          "#{message[0...i]} (#{klass})#{message[i..-1]}"
+        if klass == RuntimeError && message.empty?
```

Comment:
```suggestion
        if Primitive.equal?(klass, RuntimeError) && message.empty?
```

Reply:
Since we are going to need to test this case no matter highlight or not, maybe it's worth testing it before the `if highlight` and store it in a local variable like `empty_runtime_error = ...`

Reply:
Good point. I only handled the `highlight: true` case at first. I didn't realize the message was built up differently until after I saw the IRB tests were still failing. I should have looked at cleaning it up afterwards.